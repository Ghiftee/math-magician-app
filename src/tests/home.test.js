import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../pages/home';

test('expect home page to match working snapshot', () => {
  const homePage = renderer.create(<Home />).toJSON();
  expect(homePage).toMatchSnapshot();
});
