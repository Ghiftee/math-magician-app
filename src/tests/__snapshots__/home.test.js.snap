// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect home page to match working snapshot 1`] = `
<div>
  <h2
    style={
      Object {
        "marginBottom": "20px",
        "marginTop": "50px",
      }
    }
  >
    Welcome to our Page!
  </h2>
  <p>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis id mattis felis. Nam accumsan leo justo. In convallis tellus sit amet iaculis tincidunt. Ut quis porttitor ante. Maecenas nec suscipit ex. Vivamus non sapien nisl. Vestibulum nisi tortor, cursus at feugiat quis, consequat mattis enim. Maecenas blandit lobortis erat, eget elementum massa ultricies in. Curabitur vestibulum porttitor nisl, nec efficitur mauris lobortis a. Curabitur iaculis gravida pulvinar. Sed vel dictum metus, consectetur consectetur lorem. Maecenas tellus est, congue vitae tellus hendrerit, consequat mattis lacus. Proin a ex ac magna venenatis lacinia at a leo. Suspendisse luctus dapibus elit, sit amet varius ligula. Aliquam vitae lectus vitae massa fermentum bibendum a eu ligula.
    <br />
    <br />
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis id mattis felis. Nam accumsan leo justo. In convallis tellus sit amet iaculis tincidunt. Ut quis porttitor ante. Maecenas nec suscipit ex. Vivamus non sapien nisl. Vestibulum nisi tortor, cursus at feugiat quis, consequat mattis enim. Maecenas blandit lobortis erat, eget elementum massa ultricies in. Curabitur vestibulum porttitor nisl, nec efficitur mauris lobortis a. Curabitur iaculis gravida pulvinar. Sed vel dictum metus, consectetur consectetur lorem. Maecenas tellus est, congue vitae tellus hendrerit, consequat mattis lacus. Proin a ex ac magna venenatis lacinia at a leo. Suspendisse luctus dapibus elit, sit amet varius ligula. Aliquam vitae lectus vitae massa fermentum bibendum a eu ligula.
  </p>
</div>
`;
